server.js

```diff
@@ -10,7 +10,7 @@ app.set('views', 'templates')
 app.set('view engine', 'pug')
 
 // routes
-app.get('/', (request, response) => response.render('whiteboard', {title: 'Study Group'}))
+app.get('/', (request, response) => response.render('whiteboard/whiteboard', {title: 'Study Group'}))
```

Comment:
Now, since it's templates/whiteboard/whiteboard.pug we have to do whiteboard/whiteboard to find the template